### ISSUE TITLE
Atmospheric Tanks do not connect to adjacent pipes during mapload

### ISSUE THREAD

**Issue by newfren** [tgstation/tgstation#61594]
## Reproduction:
Observed over a couple rounds, just now remembered to report it.
https://puu.sh/IcG4X/c9fed6e28a.png

It's also "broken and nonfunctional", as is the one in the turbine, when examined.

Also, I have no idea how these even connect to pipes? I surrounded it with omni pipes and none seemed to connect.

**Comment by timothymtorres** [comment]
The air tanks probably need the `dir` variable set to the proper direction.

**Comment by Maurukas** [comment]
After some research, this is not related to the dir variable not being set. The object itself does not do anything with the dir variable. /obj/machinery/atmospherics/components/tank/ do not generate connection ports in initialize() or lateinitialize().

Tanks can still be wrenched to generate a connection to the tile a player stands on, but this is not useful for mapping. These tanks are used as air or fuel supply when mapped in, and the expectation is that they will connect to an adjacent pipe network in the appropriate direction like other atmos devices, and the storage tanks they replaced did.

This code was added in #59556, back in June.

Actual behavior in game (based on dir setting)
![2021-09-21 13_52_20-Space Station 13](https://user-images.githubusercontent.com/66576896/134246332-49ec7502-8e48-4968-a8bd-b4795ba3f44b.png)

Expected behavior (based on dir settings):
![2021-09-21 13_54_00-Space Station 13](https://user-images.githubusercontent.com/66576896/134246361-fb430195-53a3-40c4-8682-26b290d18b41.png)

**Comment by LemonInTheDark** [comment]
@esainane this smells like something related to your recent work. I know it worked properly at one point

**Comment by esainane** [comment]
I checked out the commit immediately before #60981 was merged. Mapping in a smart pipe immediately east of the plasma pressure tank in the MetaStation incinerator does not result in a connection then, either. Wrenching it as the examine text suggests ("A pipe port can be opened with a wrench.") does work.

`git blame` suggests the last major change with pressure tanks was #59556. Checking out the revision immediately before, the pressure tank has a port out, even without mapping changes. Looking at this PR, it seems the type path moved to no longer be a subtype of `unary/`, so it no longer gets a default port.

The additional examine text was also added in this PR, so perhaps this was intended behavior of the PR? It seems like there should still be a way for mappers to add roundstart connected pressure tanks, though.
